### PR TITLE
Meet the team

### LINKED ISSUE #6: Create a new webpage called "Meet the Team"
- [x] I personally would go about doing this by making a copy of the live folder and replacing the word live with "Meet the Team" everywhere on the website
- [x] We wouldn't need resources, tracks/prizes or schedule folder/tab on the website so please delete that from the live folder copy and you should be left with only an index.html file and a .DS_Store file which is just a binary file we don't need to bother about right now.
- [x] On the website, clear out all information that was earlier on the live page. Keep the link that takes us back to the landing page.
- [x] Replace the three lilac divs with attribute: [class="card full-width"] with two card full-width lilac divs where we will later put in images of directors and members respectively.

You would be left with a pretty empty website with no information but just a tab called "Meet the Team!", two lilac divs and all necessary formatting(the background for example) remains intact.

### LINKED ISSUE #7: Meet The Team Page - Create Team Member Carousel
Create a carousel component for this page that has a left and right button or arrow to scroll through and see all the team members.  Each tile in the carousel should have an image followed by the name and role underneath them (feel free to get creative with this).

Here is a youtube tutorial that could help with this, but feel free to use a different tutorial if you would like. 
https://www.youtube.com/watch?v=9HcxHDS2w1s 

NOTE: You don't need to add any actual information to the tiles, just format them.

Requirements:
The carousel is visible with tiles and scrollable through some form of a button or an arrow, and each tile has space for an image, name, and role.

### PR DESCRIPTION
**Pull request by vvkash**
Added Meet The Team page and Link to it in the nav bar, also changed the file location.

Had to make another pull request, the other one was having issues lol

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
             <li id="nav-section"><a href="#about">About</a></li>
             <li id="nav-section"><a href="#faq">FAQ</a></li>
             <li id="nav-section"><a href="#sponsors">Sponsors</a></li>
+            <li id="nav-section"><a href="#meettheteam">Meet The Team</a></li>
```
Comment (rishyendra333) [inline index.html]: change the href from #meettheteam to /meet-the-team/index.html to link to that page.

**Review by rishyendra333 — CHANGES_REQUESTED** [review]

**Comment by aryasalian** [comment]
I went over your code and fixed what rishi had asked for. There are still some issues with it and I tried fixing it as much as possible but I'll let you take over it again now. I noticed the slider would go way beyond where it's supposed to end so I tried to make some edits to your slider. There is a considerable amount of improvement but I need you to look into why the transformX isn't working accurately at different window sizes. It sometimes cuts the images or overshoots the scroll even though I've tried reducing that issue as much as possible. Focus on the line that causes the scroll, it's to do with the math of it.

**Comment by vvkash** [comment]
Yeah i see, i totally mislooked that haha. im refactoring right now and i think i got the issue where it overshoots and the issue with different window sizes. though for some reason the updated code wont allow me to click the arrows to scroll through, ive checked the event listeners as well. If its okay with everyone, i can implement all the changes you've done and the new updates to the sizing and issues but keep some of the arrow functionality and then commit again? lmk if this works and ill get it done and commit it this week!

**Comment by vvkash** [comment]
Also just to make sure, you want to keep the picture sizing the same even when the window itself is being changed right? also is there a certain sizing guide you want me to impliment and follow for the slider? Thanks!

**Comment by aryasalian** [comment]
For the arrow functionality, it works as it is supposed to when I put my commits in...it is not supposed to scroll ahead if the last image in the slider is visible and at a normal window size, since we only have 3 images in the slider currently, the third image is visible in the slider by default so it seems that the arrow doesn't work but if you try and shrink the window to only keep one or two or even two and a fraction image visible, the scrolling functionality will work. It's the math of how many times should you be allowed to click the arrow for scrolling to the end is what needs to be fixed because as I said, when I was working with it, it either overshot it or didn't let you scroll enough.

As for the image size I think it is smart to have the image shrink to scale with window size since that would make all this much easier and we can instead just code each slider's arrow to be allowed to be clicked a fixed number of times instead of making it dependent on the no. of visible images/total images in the slider. Ensure that after a certain small window size, the slider just disappears entirely since having super tiny images wouldn't help anyways

**Comment by aryasalian** [comment]
Good work on most of the stuff tho!

**Comment by aryasalian** [comment]
Seems good to me now but the button for junior director slider just moves with the images lol. Try and fix that. The meet the team tab on the landing page needs to be resized when the window resized. Please add a commit for that in this branch itself. Make sure the resizing works for the mobile-view's hamburger dropdown as well. Everything else looks great!

**Comment by vvkash** [comment]
Fixed The Issue with the slider arrow and will work on adding the logo for the home link and adding the arrows to the border of the sliders in the next few days. Been out of town so havent been able to finish that! Also will fix the resizing issue regarding the meet the team logo and the hamburger menu by next  week as well.

**Pull request merged**